API/chat/tests.py

```diff
@@ -695,7 +723,7 @@ def test_message_create(self):
 
         dbmessage = messages[0]
 
-        self.assertEqual(dbmessage.text, message.text)
+        self.assertEqual(dbmessage.message_content, message.message_content)
         self.assertEqual(dbmessage.datetime_start, message.datetime_start)
         self.assertEqual(dbmessage.username, message.username)
         self.assertEqual(dbmessage.channel.id, message.channel.id)
```

Comment:
Add some tests to ensure the message_type 'image' is accepted. Maybe change some existing tests to use image instead of text?